### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [ForsakenDelusion/AUST-CS-Plan#4] [Announcements]
# index

AUST计算机求生指南

https://aust-cs-plan.forsakendelusion.online/

**Comment by ForsakenDelusion** [comment]
hi

**Comment by ForsakenDelusion** [comment]
我补药学习啊😭

**Comment by iocion** [comment]
怎么没有人哇

**Reply by ForsakenDelusion** [reply]
刚放暑假 大🔥都在玩呢嘛 有空再更新一下

**Comment by GaIIagherdev** [comment]
乔不思前来资瓷! ![](https://cdn-app-icon2.pgyer.com/1/b/3/2/7/1b327979336668a3381b38a0bdcf38db?x-oss-process=image/resize,m_lfit,h_120,w_120/format,jpg)

**Reply by ForsakenDelusion** [reply]
好耶😍

**Reply by iocion** [reply]
乔不思大佬来了**厉害**

**Comment by ProgramCX** [comment]
校友来支持😍
